elements/pl-multiple-choice/pl-multiple-choice.py

```diff
@@ -364,11 +364,18 @@ def test(element_html, data):
     result = data['test_type']
     if result == 'correct':
         data['raw_submitted_answers'][name] = data['correct_answers'][name]['key']
-        data['partial_scores'][name] = {'score': 1, 'weight': weight}
+        for option in data['params'][name]:
+            if option['key'] == correct_key:
+                feedback = option['feedback']
+        data['partial_scores'][name] = {'score': 1, 'weight': weight, 'feedback': feedback}
```

Comment:
```suggestion
        feedback = data['correct_answers'][name].get('feedback', None)
        data['partial_scores'][name] = {'score': 1, 'weight': weight, 'feedback': feedback}
```

I'm not 100% sure this would work, but you should test it.

Reply:
I tested this locally and it appears to work. I'll go ahead and commit it.